### ISSUE TITLE
Voting UI redesign

### ISSUE THREAD

**Issue by sinisarudan** [Cha-OS/KnAllEdge#234]
current UX is not intuitive neither has a "call-for-action"
![screen shot 2016-07-29 at 5 07 28 pm](https://cloud.githubusercontent.com/assets/1769627/17252963/fdd39b74-55ae-11e6-998a-12a7b843839f.png)
in current UX, clicking on upper area causes for vote-up, on the lower vote-down
in the same time upper area represents user's votes, and lower total votes.

**Comment by mprinc** [comment]
Yes, @sinisarudan i think we should redesign it. Most likely I will start with refactoring halo, to make it in one line, also I will make different organization and interface of creating node-flags (like voting, slice, approval, ...) and then I will deal with this.

**Comment by mprinc** [comment]
I have already created IBIS form so there there are already vote up/down icons with thumbs
